### ISSUE TITLE
Breaking changes in bioregistry

### ISSUE THREAD

**Issue by bgyori** [biopragmatics/biomappings#78]
Looks like https://github.com/biopragmatics/bioregistry/pull/213 made breaking changes to a public API method of the Resolver class in the Bioregistry causing failures here.

**Comment by bgyori** [comment]
It appears, the latest changes also changed the expected identifier format, inconsistent with the MIRIAM standard (where available). It is my position that the resource files in this repository should retain their existing standard therefore I made changes accordingly in https://github.com/biopragmatics/biomappings/pull/77.

**Comment by bgyori** [comment]
Reopening due to more breaking changes:
https://github.com/biopragmatics/biomappings/runs/4199950238?check_suite_focus=true
```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.8/x64/bin/biomappings", line 33, in <module>
    sys.exit(load_entry_point('biomappings', 'console_scripts', 'biomappings')())
  File "/opt/hostedtoolcache/Python/3.9.8/x64/lib/python3.9/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.9.8/x64/lib/python3.9/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/opt/hostedtoolcache/Python/3.9.8/x64/lib/python3.9/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.9.8/x64/lib/python3.9/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/hostedtoolcache/Python/3.9.8/x64/lib/python3.9/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/runner/work/biomappings/biomappings/src/biomappings/upload_ndex.py", line 45, in ndex
    cx.set_context({prefix: bioregistry.get_format_url(prefix) for prefix in prefixes})
  File "/home/runner/work/biomappings/biomappings/src/biomappings/upload_ndex.py", line 45, in <dictcomp>
    cx.set_context({prefix: bioregistry.get_format_url(prefix) for prefix in prefixes})
AttributeError: module 'bioregistry' has no attribute 'get_format_url'
Error: Process completed with exit code 1.
```

**Comment by cthoyt** [comment]
Solved in 35399c5. Sorry you had to figure this all out, it was all sitting uncommitted on my local machine :/
